inst/data_scripts/rs.R

```diff
@@ -0,0 +1,186 @@
+# RS
+
+library(dplyr)
+library(admiral)
```

Comment:
Could you add `library(tidyselect)`? Otherwise the script fails when `all_of()` is called.

Reply:
Added tidyselect to all three scripts (tu, tr, rs).